app/views/orga/teams/show.html.slim

```diff
@@ -79,7 +79,7 @@ nav.btn-group
   ul.conferences
     - @team.members.each do |user|
       - if user.conferences.in_current_season.any?
-        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season, true).join(',')}".html_safe
+        li = "#{link_to user.name, user}: #{links_to_conferences(user.conferences.in_current_season).join(',')}".html_safe
```

Comment:
Since the helper method is already doing quite a lot, I would suggest to also move `join` and `html_safe` there. It looks kind of weird to me, having a rather long method called `links_to_conferences` not returning the actual links in a *"view-ready*" format.

What do you think?

It would also make this list item a bit cleaner:
```ruby
li = "#{link_to user.name, user}: #{link_to_conferences(user.conferencs.in_current_season)}
```